### ISSUE TITLE
Approval Submissions - Decline?

### ISSUE THREAD

**Issue by mharris9180** [F17MIS4173Team3/Apex#21]
Should there be a decline option for approval submissions?

**Comment by jasongreb** [comment]
Yes! There should... for the moment, you'd just ignore it. I suppose we need to decide what happens when they decline it and how it would get rectified by the user. This would certainly be some new build.

**Comment by de1417** [comment]
A denied/rejected entry could potentially notify the user via email that entry <value> was declined for <reason>, and resubmit or contact HR with questions.  Reason could be a text field populated by HR staff prior to selecting the deny/reject button.
